packages/api/cms-api/src/user-permissions/access-control.service.test.ts

```diff
@@ -3,6 +3,12 @@ import { Test, type TestingModule } from "@nestjs/testing";
 import { AbstractAccessControlService } from "./access-control.service";
 import { type CurrentUser } from "./dto/current-user";
 
+declare module "./user-permissions.types" {
+    export interface PermissionOverrides {
+        accessControlServiceTest: "p1" | "p2";
+    }
+}
+
```

Comment:
that's the con for module augmentation: this effects all tests, even the whole comet (although it is not exported of course)

Reply:
Could we use as casts for these tests instead of module augmentation?

Reply:
yes we could but this will be many casts 🙈

Reply:
solved with 0da33fe5a77a06cb8f392d6e8b031b4fc45ce3cc